src/containers/AboutUs/index.js

```diff
@@ -0,0 +1,268 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled, { withTheme } from 'styled-components';
+import Button from '@material-ui/core/Button';
+import ArrowDownwardIcon from '@material-ui/icons/ArrowDownward';
+
+import FounderCard from '../../components/UI/FounderCard';
+import JoinUsImage from '../../static/images/backgrounds/join-us.jpg';
+import HookedParallax from '../../components/UI/HookedParallax';
+
+const AboutUs = (props) => {
+  // Hooks
+  const founderContainerRef = useRef(null);
+
+  // Functions
+  const ArrowDownClickHandler = () => {
+    if (founderContainerRef && founderContainerRef.current) {
+      founderContainerRef.current.scrollIntoView({
+        behavior: 'smooth',
+        block: 'end',
+        inline: 'nearest'
+      });
+    }
+  };
+
+  const foundersData = [
```

Comment:
We can use `useMemo` for this array for optimization.

Reply:
fixed